### ISSUE TITLE
NcSelect is not rendered correctly

### ISSUE THREAD

**Issue by calvinrw** [nextcloud/nextcloud-vue#3593]
# Expected behavior

An `NcSelect` component will be rendered like shown in the [Vue style guide](https://nextcloud-vue-components.netlify.app/#/Components/NcSelect), where its text input, drop-down menu, clear button, and drop-down button are in the same positions and are enclosed by a thick border.

No option selected / drop-down open:

<img width="419" alt="Screen Shot 2022-12-24 at 10 12 54" src="https://user-images.githubusercontent.com/687129/209443781-941c28f1-93b1-4390-acb3-75fa57b322b7.png">

Option selected:

<img width="413" alt="Screen Shot 2022-12-24 at 10 13 11" src="https://user-images.githubusercontent.com/687129/209443790-d7b6d2c4-fa2c-466a-9541-ce8e66079f9b.png">

# Actual behavior

When the minimum reproducible example that follows is compiled and rendered, `NcSelect`'s sub-components' style and positioning do not match what's shown in the documentation and are presumably incorrect.

No option selected / drop-down open:

<img width="293" alt="Screen Shot 2022-12-24 at 10 10 33" src="https://user-images.githubusercontent.com/687129/209443705-1f0665b8-5c50-4dbb-8754-71bdb4b94c38.png">

Option selected:

<img width="259" alt="Screen Shot 2022-12-24 at 10 11 04" src="https://user-images.githubusercontent.com/687129/209443739-3704c631-0eb6-42a6-ae14-8a17609b5f95.png">

## Minimum reproducible example
```vue
<template>
    <div class="test-view">
        <NcSelect placeholder="Select a thing"
                  v-bind="selectOptions"
                  v-model="selectValue"/>
    </div>
</template>

<script>
import NcSelect from "@nextcloud/vue/dist/Components/NcSelect"

export default {
  components: {
    NcSelect,
  },
  
  data() {
    return {
      selectValue: null,

      selectOptions: {
        inputId: 'select-1',
        options: [
          'Option 1',
          'Option 2',
          'Option 3',
          'Option 4',
        ],
      },
    }
  },
  
}
</script>

<style>
.test-view {
    padding: 50px;
}
</style>

```

Potentially relevant bits from `package.json` for my project:
```js
  "dependencies": {
    "@nextcloud/axios": "^2.3.0",
    "@nextcloud/dialogs": "^3.2.0",
    "@nextcloud/router": "^2.0.0",
    "@nextcloud/vue": "^7.3.0",
    "vue": "^2.7.14",
    "vue-good-table": "^2.21.11",
    "vue-material-design-icons": "^5.1.2",
    "vue-router": "^3.6.5",
    "vuex": "^3.6.2",
    "vuex-router-sync": "^5.0.0"
  },
  "browserslist": [
    "extends @nextcloud/browserslist-config"
  ],
  "engines": {
    "node": "^16.0.0",
    "npm": "^7.0.0 || ^8.0.0"
  },
  "devDependencies": {
    "@nextcloud/babel-config": "^1.0.0",
    "@nextcloud/browserslist-config": "^2.3.0",
    "@nextcloud/eslint-config": "^8.1.4",
    "@nextcloud/stylelint-config": "^2.3.0",
    "@nextcloud/webpack-vue-config": "^5.4.0",
    "webpack-cli": "^4.10.0"
  }
```

**Comment by Pytal** [comment]
Does this occur on the latest `master` of https://github.com/nextcloud/nextcloud-vue and https://github.com/nextcloud/server @calvinrw?

**Comment by dartcafe** [comment]
I can confirm, it occurs with the latest release. Master not tested.

**Comment by raimund-schluessler** [comment]
Works fine with 7.7.1 as far as I can tell:
<img width="225" alt="NcSelect" src="https://user-images.githubusercontent.com/2496460/221600226-c76cb420-d21a-4cdd-bf6b-ba21ee88b98b.png">

**Comment by calvinrw** [comment]
I can confirm that this is fixed in 7.7.1. Thanks for all your hard work!
